rc/base/autowrap.kak

```diff
@@ -37,9 +39,9 @@ def -hidden autowrap-cursor %{ eval -save-regs '/"|^@m' %{
 } }
 
 def autowrap-enable -docstring "Wrap the lines in which characters are inserted" %{
-    hook -group autowrap window InsertChar [^\n] autowrap-cursor
+    hook -group autowrap buffer InsertChar [^\n] autowrap-cursor
```

Comment:
That's the thing, when I thought about it, I it looked like I would typically want a file to have it's contents wrapped, not having two windows open on the same file and have only one that has wrapping activated.